molecule/driver/openstackdriver.py

```diff
@@ -35,6 +35,30 @@ def __init__(self, molecule):
         self._provider = self._get_provider()
         self._platform = self._get_platform()
         self._openstack = shade.openstack_cloud()
+        self._keypair_name = self.get_keypair_name()
+        self._molecule_generated_ssh_key = False
+
+    def get_keyfile(self):
+        # allow python to access the home directory
+        from os.path import expanduser
+        home = expanduser("~")
+        fileloc = home + "/.ssh/id_rsa"
```

Comment:
Should you test for the public key file here, since ultimately that is what you want to use?